test/stress_tests/run_application_stress_tests.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+# This script runs all of the application tests.
+
+
+ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
+RAY_VERSION=$(git describe --tags --abbrev=0)
+RESULT_FILE="results-$(date '+%Y-%m-%d_%H-%M-%S').log"
```

Comment:
Could we write this in python? I have a feeling this script will get more complicated over time and it would become hard to maintain in bash.

Also, it is easier to support parallel runs in Python.